### DISCUSSION TITLE
Looking for extension for react and html autocomplete

### DISCUSSION THREAD

**Discussion by shahbaz4783** [zed-industries/zed#17255] [Feature Requests]
Is there any extension which can give me boilerplate functional component like the vscode "rfce"?
Also the html tag doesnt auto close, i have to close the tag manually. Is there any configuration i need to do?

**Comment by abena07** [comment]
following

**Comment by maiquealmeida** [comment]
following

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
